src/main/java/org/logstash/beats/Server.java

```diff
@@ -134,6 +134,7 @@ public void initChannel(SocketChannel socket){
                              new IdleStateHandler(localClientInactivityTimeoutSeconds, IDLESTATE_WRITER_IDLE_TIME_SECONDS, localClientInactivityTimeoutSeconds));
             pipeline.addLast(BEATS_ACKER, new AckEncoder());
             pipeline.addLast(CONNECTION_HANDLER, new ConnectionHandler());
+            pipeline.addLast(new FlowLimiterHandler());
```

Comment:
Do we have a reason why this is placed in this location of the pipeline? would it make sense to be first instead? Or right after the ssl handler?

Reply:
Moved in the same position it was from the originating PR #475, but your point is correct. It should be the first handler in the Netty pipeline